### ISSUE TITLE
Trm1022a corrections

### ISSUE THREAD

**Issue by craigsapp** [TassoInMusicProject/tasso-website#85]
"Prima parte," Seconda parte, " etc. (this piece is in four parts) should
go above the staff, not below.

The last note of each section should be a long as if it were the end of a
piece (or a fermata, whichever we are doing at the end of pieces)

**Comment by craigsapp** [comment]
> "Prima parte," Seconda parte, " etc. (this piece is in four parts) should
go above the staff, not below.

These are now fixed.   When encoding a work containing "Prima parte", "Seconda parte", etc., prefix the name of the section with the text `section: `.  This will allow the converter to recognize the text as a section name.

Here is the fixed example:

http://www.tassomusic.org/work/?id=Trm1022a

The first section is labeled properly:

<img width="1076" alt="screen shot 2018-03-08 at 7 44 04 pm" src="https://user-images.githubusercontent.com/3487289/37189389-fa34d5c2-2308-11e8-9fba-2aac9945fd80.png">

The second part, etc. are not being labeled:

<img width="1079" alt="screen shot 2018-03-08 at 7 45 45 pm" src="https://user-images.githubusercontent.com/3487289/37189394-005de51a-2309-11e8-8bb3-cf889c55daf3.png">

In this case the data file is correct, but I will have to adjust the converter to have it display properly.

> The last note of each section should be a long as if it were the end of a
> piece (or a fermata, whichever we are doing at the end of pieces)

Notice in the above example this is now fixed.   Longs are displaying properly in verovio it looks like, so I added back the terminal longs, replacing the fermatas.  In the conversion of the initial MusicXML files, the ends of sections seem to be final barlines.  I changed them to double barlines (I will check the MusicXML files to see if they are actually supposed to be final barlines).

**Comment by emilianoricciardi** [comment]
Sounds good. For longs, we also need to figure out what to do when they are
not aligned across the voices. For example, in Trm0248d, the long in the
Alto begins in the penultimate measure (maybe add/tie another long in the
last measure? or remove bar line in Alto part? what did you do for similar
situations in JRP?)

Emiliano

On Thu, Mar 8, 2018 at 10:46 PM, Craig Stuart Sapp <notifications@github.com

**Comment by craigsapp** [comment]
In JRP style, the barline should be removed.

Current view on TiMP for Trm0248d:

http://www.tassomusic.org/work/?id=Trm0248d

<img width="633" alt="screen shot 2018-03-08 at 8 26 06 pm" src="https://user-images.githubusercontent.com/3487289/37190467-c8509130-230e-11e8-9b74-9c746ac1e8eb.png">

The proper display would be to hide the barline:

<img width="633" alt="screen shot 2018-03-08 at 8 26 06 pm" src="https://user-images.githubusercontent.com/3487289/37190549-3d776772-230f-11e8-91d7-df6481e78ff6.png">

I don't think that the barline can be made invisible in verovio yet, though...

**Comment by emilianoricciardi** [comment]
Yes, hiding the barline would be ideal. How did you create the second
sample if Verovio can't hide the barline?

On Thu, Mar 8, 2018 at 11:30 PM, Craig Stuart Sapp <notifications@github.com

**Comment by craigsapp** [comment]
> Yes, hiding the barline would be ideal. How did you create the second
> sample if Verovio can't hide the barline?

I did that in Photoshop.

Here is the request for this feature to be added into MEI:

https://github.com/music-encoding/music-encoding/issues/470

After it is in MEI, then it would have to be implemented in verovio...

**Comment by emilianoricciardi** [comment]
OK, they'll have to thank us for raising all these issues!

Sticking to Trm1022a, I think the Alto part should be in the lower-octave
treble clef...too many extra lines at the moment. What do you think?

On Thu, Mar 8, 2018 at 11:39 PM, Craig Stuart Sapp <notifications@github.com

**Comment by craigsapp** [comment]
> Sticking to Trm1022a, I think the Alto part should be in the lower-octave
treble clef...too many extra lines at the moment. What do you think?

Yes, it would definitely look better in vocal-tenor clef.  I just changed it (but you will need to close all of the TiMP website pages and reopen to see the change).

http://www.tassomusic.org/work/?id=Trm1022a


<img width="1023" alt="screen shot 2018-03-08 at 9 17 25 pm" src="https://user-images.githubusercontent.com/3487289/37191643-c4c26e24-2315-11e8-856e-626cf4e9c4c0.png">
